glossary.yml

```diff
@@ -157,6 +157,24 @@ entries:
     description: |-
       Consolidating two or more requirements for supplies or services previously provided or performed under separate smaller contracts, into a solicitation for a single contract that is likely to be unsuitable for award to a small business concern.
 
+  BD:
+    type: acronym
+    term: Blanket purchase agreement
```

Comment:
@c-collins Looks like this might be linking to the wrong term, I suspect it should be `Business development`

Reply:
@beechnut you are correct. Good catch.